src/api_server.h

```diff
@@ -27,15 +27,15 @@ namespace http = boost::network::http;
 namespace google {
 
 // Configuration object.
-class MetadataAgentConfiguration;
+class Configuration;
 
 // Storage for the metadata mapping.
 class MetadataStore;
 
 // A server that implements the metadata agent API.
 class MetadataApiServer {
  public:
-  MetadataApiServer(const MetadataAgentConfiguration& config,
+  MetadataApiServer(const Configuration& config,
                     const MetadataStore& store, int server_threads,
```

Comment:
Sorry, didn't notice this before. Same here: `const MetadataStore& store` fits on the previous line. In fact, this can become:
```cpp
  MetadataApiServer(const Configuration& config, const MetadataStore& store,
                    int server_threads, const std::string& host, int port);
```

Reply:
Fixed.